### PR TITLE
Refresh NFTs collections improvements

### PR DESCRIPTION
**Pull request by jagodarybacka**
Resolves https://github.com/tallyhowallet/extension/issues/2896

### What

This PR introduces improvements on reloading NFTs:
- Reload properly on transfers between accounts loaded in the wallet
  - Testing: see linked issue - try transferring NFTs from this collection (current test wallet) between accounts. Try transferring NFT from one account to the account that is not currently in the wallet. Check if both accounts are showing correct state.
- Refresh the metadata button on the NFT preview 
  - Testing: click Refresh metadata button on the NFT from small collection (<50 NFTs owned) and on the large collection (>50 NFTs owned). Close preview and see if collection is properly loaded, for large collection check if all NFTs are loading properly as you scroll.
image

## Testing Env

```
SUPPORT_NFT_TAB=true
```

Latest build: [extension-builds-2935](https://github.com/tallyhowallet/extension/suites/10683124426/artifacts/534556913) (as of Tue, 31 Jan 2023 10:18:34 GMT).

### CODE REVIEW

File: ui/components/NFTS_update/NFTPreview.tsx [hunk 1]
```diff
@@ -168,11 +188,21 @@ export default function NFTPreview(props: NFTWithCollection): ReactElement {
...
-            <div className="preview_section_header">
-              {t("preview.properties")}
-            </div>
+        <div className="preview_section">
+          <div className="preview_section_header">
+            {t("preview.properties")}
+          </div>
+          <SharedButton
+            type="tertiaryWhite"
+            iconSmall="swap"
```
Comment (0xDaedalus) [inline ui/components/NFTS_update/NFTPreview.tsx:197]: Don't think the icon should be `swap` here.

**Review by 0xDaedalus — APPROVED** [review]
LGTM!  Just one non-blocking thing that can be resolved in a later PR.

File: ui/components/NFTS_update/NFTPreview.tsx (hunk 1 above)
Comment (jagodarybacka) [reply ui/components/NFTS_update/NFTPreview.tsx:197]: But it is swap icon in Figma 😄 cc @VladUXUI 
image

Comment (VladUXUI) [reply ui/components/NFTS_update/NFTPreview.tsx:197]: Right riiight, i knew i was forgetting something :)) will create an icon tomorrow morning

**Pull request merged**